guides/common/modules/proc_provisioning-ubuntu-20-04-3-plus-through-smart-proxies.adoc

```diff
@@ -0,0 +1,36 @@
+[id="Provisioning_Ubuntu_20_04_3_plus_Through_Smart_Proxies_{context}"]
+= Provisioning Ubuntu 20.04.3+ Through {SmartProxies}
+
+Perform these steps to provision hosts running Ubuntu 20.04.3+ or Ubuntu 22.04.
+
+.Procedure
+. On your {SmartProxyServer}, ensure _Smart Proxy HTTP forward_ is enabled:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# grep "^:enabled: true$" /etc/foreman-proxy/settings.d/templates.yml
+----
++
+Note that this is enabled by default.
+. On your {SmartProxyServer}, open the corresponding port for the Smart Proxy service:
```

Comment:
I am unsure if we can/should say "Smart Proxy service" here.

Reply:
I get what you mean, service is a term which makes me think more about a daemon and that wouldn't fit too well here.
Instead, we could say:
```suggestion
. On your {SmartProxyServer}, open the corresponding port so the Smart Proxy can process incoming requests.
```

Reply:
Removed "for the Smart Proxy service" instead.